docs/content/app-protect/configuration.md

```diff
@@ -19,7 +18,8 @@ The NGINX Ingress Controller has a set of global configuration parameters that a
 
 ## Enable App Protect for an Ingress Resource
 
-You can enable and configure NGINX App Protect on a per-Ingress-resource basis. To do so, you can apply the [App Protect annotations](/nginx-ingress-controller/configuration/ingress-resources/advanced-configuration-with-annotations/#app-protect) to each desired resource.
+You can enable and configure NGINX App Protect on the Custom Resources (VirtualServer, VirtualServerRoute) or on the Ingress-resource basis.
```

Comment:
This should link to the Custom Resource examples.
If this is not documented, it needs to be.

It is lightly documented here: https://docs.nginx.com/nginx-ingress-controller/configuration/policy-resource/#waf

Reply:
done